<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crud::UserView</class>
 <widget class="QWidget" name="crud::UserView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Datos del Usuario</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nombre </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="userName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="loginName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Contrasena</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="email">
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Privilegios</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabOrgPrivileges">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="bSelectAll">
           <property name="text">
            <string>Todos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bSelectNone">
           <property name="text">
            <string>Ninguno</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="bExpandAll">
           <property name="text">
            <string>Expandir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bCollapseAll">
           <property name="text">
            <string>Contraer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userName</tabstop>
  <tabstop>loginName</tabstop>
  <tabstop>password</tabstop>
  <tabstop>email</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>bSelectAll</tabstop>
  <tabstop>bSelectNone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
